Tensile/Source/lib/source/ocl/OclSolutionAdapter.cpp

```diff
@@ -232,7 +232,7 @@ namespace Tensile
                         {
                             std::cerr << "KernelError: " << e.err() << std::endl;
                         }
-                        throw e;
```

Comment:
cppcheck: Throwing a copy of the caught exception instead of rethrowing the original exception.